plugins/cli-plugins/index.js

```diff
@@ -0,0 +1,31 @@
+module.exports = cliPlugins;
+
+function cliPlugins() {
+    return {
+        name: require('./package.json').name,
+        plugins: [
+            testOne(),
+            testTwo()
+        ]
+    };
+}
+
+function testOne() {
+    return {
+        name: 'hello',
+        description: 'testing hello plugin',
+        action: () => {
+            console.log('hello');
+        }
+    };
```

Comment:
How about the following?
```js
return {
  cliCommands: [
    {
      name: 'hello',
      description: 'testing hello plugin',
      action: () => {
        console.log('hello');
      },
    },
  ],
}
```
A plugin may want to add several commands and also do other things beyond adding cli commands

Reply:
One problem with this, though, is that because of the way get_r_objects and retrieve_r_objects works, it must have a "name" field or it throws an error.  So adding that extra layer of cliCommands causes it to throw an error.

Reply:
Ah yes didn't think of the name of the plugin. So something like the following then.

```js
return {
  cliCommands: [
    name: 'name-of-the-plugin',
    {
      name: 'name-of-the-command',
      description: 'testing hello plugin',
      action: () => {
        console.log('hello');
      },
    },
  ],
}
```

Reply:
Then cliCommands couldn't be an array, right?  Or did you want it to be an array of objects?  Or did you want something like this:

```javascript
return {
  cliCommands: {
    name: 'name-of-the-plugin',
    plugin: {
      name: 'name-of-the-command',
      description: 'testing hello plugin',
      action: () => {
        console.log('hello');
      },
    },
  },
}
```

Reply:
Ah sorry I meant
```js
return {
  name: 'name-of-the-plugin',
  cliCommands: [
    {
      name: 'name-of-the-command',
      description: 'testing hello plugin',
      action: () => {
        console.log('hello');
      },
    },
  ],
};
```

Reply:
In other words a single plugin can add several commands